src/reference/asciidoc/channel.adoc

```diff
@@ -201,6 +203,16 @@ CAUTION: The sender can sometimes block.
 For example, when using a `TaskExecutor` with a rejection policy that throttles the client (such as the `ThreadPoolExecutor.CallerRunsPolicy`), the sender's thread can execute the method any time the thread pool is at its maximum capacity and the executor's work queue is full.
 Since that situation would only occur in a non-predictable way, you should not rely upon it for transactions.
 
+[[flux-message-channel]]
+===== `FluxMessageChannel`
+
+The `FluxMessageChannel` is a `org.reactivestreams.Publisher` implementation for `"sinking"` sent messages into an internal `reactor.core.publisher.Flux` for on demand consumption by the reactive subscribers downstream.
+This channel implementation is nor a `SubscribableChannel`, neither `PollableChannel`, so only `org.reactivestreams.Subscriber` instances can be used for this channel consumption honoring back-pressure nature of reactive streams.
```

Comment:
is an `org...

by reactive subscribers downstream.

neither a ``, nor a ``

can be used to consume from this channel